addons/actions/README.md

```diff
@@ -13,8 +13,8 @@ Import the `action` function and use it to create actions handlers. When creatin
 > *Note: Make sure NOT to use reserved words as function names. [issues#29](https://github.com/storybooks/storybook-addon-actions/issues/29#issuecomment-288274794)*
 
 ```js
-import { storiesOf, action } from '@storybook/storybook'
-// or import { action } from '@storybook/storybook-addon-actions'
+import { storiesOf, action } from '@storybook/react'
```

Comment:
FYI all of these are going to conflict with the changes I made in https://github.com/storybooks/storybook/issues/1017